flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -140,11 +142,11 @@ under the License.
 			<artifactId>scala-library</artifactId>
 			<scope>provided</scope>
 		</dependency>
-		<dependency>
+<!--		<dependency>
```

Comment:
No, actually it is a test leftover. I wanted to know the guava version of the new driver transitive dependency. I forgot to enable the dep back. I prefer having the explicit dep as the code refers it rather than relying on the transitive dep.